### PR TITLE
r.in.wms: gdal user:password support

### PR DESCRIPTION
**Pull request by AnikaBettge**
- add wms version 1.1.0
- add user:password to GDAL driver version

### CODE REVIEW

**Comment by AnikaBettge** [comment]
Added debugging the WMS request xml and if for the case that there are no user and password set.

File: scripts/r.in.wms/wms_gdal_drv.py [hunk 1]
```diff
@@ -137,11 +141,14 @@ def _download(self):
                             "geographic projection, \n try 'WMS_GRASS' driver or use WMS version 1.1.1."))
 
         self._debug("_download", "started")
-
         temp_map = self._tempfile()
 
         xml_file = self._createXML()
 
+        # print xml file content for debug level 1
+        file = open(xml_file, "r")
```
Comment (neteler) [inline scripts/r.in.wms/wms_gdal_drv.py]: Is it ok to `open()` "anyway" even when only needed in debug mode?

File: scripts/r.in.wms/wms_gdal_drv.py [hunk 2]
```diff
@@ -137,11 +141,14 @@ def _download(self):
...
+        grass.debug("WMS request xml:\n%s" % file.read(), 1)
```
Comment (neteler) [inline scripts/r.in.wms/wms_gdal_drv.py]:
```suggestion
        grass.debug("WMS request XML:\n%s" % file.read(), 1)
```

**Comment by neteler** [comment]
Backported to relbranch78 in 78dc9c941ae1213fa8baca9f670f6492f4e4a8ab

**Pull request merged**